app/models/condition.rb

```diff
@@ -10,10 +10,8 @@ class Condition < ApplicationRecord
 
   def save_and_update_form
     save!
-    # TODO: https://trello.com/c/dg9CFPgp/1503-user-triggers-state-change-from-live-to-livewithdraft
-    # Will not be needed when users can trigger this event themselves through the UI
-    form.create_draft_from_live_form if form.live?
```

Comment:
Huh, I didn't even notice this bug before, saving a condition didn't switch `archived` to `archived_with_draft`. That's a plus for this commit reducing repetition!